app/javascript/components/Backlinks.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react'
 
-function Backlinks({ backlinks }) {
+export default function Backlinks({ backlinks }) {
```

Comment:
I think we should avoid using default exports

Reply:
Summary of discussion with @dennyluan in person.

- stop using `export default` from here forward.

todo: 
- [ ] remove `default` from all instances of `export default`
- [ ] fix all instances where the components are imported